doc/tensor.md

```diff
@@ -1327,36 +1331,64 @@ x:indexFill(2,torch.LongTensor{4,2},-10)
 ```
 
 <a name="torch.Tensor.gather"/>
-### [Tensor] gather(src, dim, index) ###
+### [Tensor] torch.gather(src, dim, index) ###
```

Comment:
No worries, keep 'em coming :-)

I put the `torch.` in there to try and make it _more_ consistent, but it must have backfired! My reasoning was that the other functions are mostly documented for the calling pattern `src:func(param1, param2)` whereas this one is documented with the `src` tensor as an argument which is more like the `torch.func(src, param1, param2)` calling pattern (or the `src:func(src, param1, param2)` pattern, which doesn't make much sense for this function). My first instinct was to remove the `src` parameter from the heading, but then there's no name to refer to the `src` tensor :-(

The reason I changed it is because I found the existing docs confusing - before I understood the function I thought `src` was a *third* tensor involved somehow (like `x:gather(src, 1, i)). I defer to your judgment as to the best way to present it. Let me know what you think and I'll make the change.